### PR TITLE
Reduce required boilerplate deletion in PR template.

### PR DESCRIPTION
**Pull request by jrouly**
Forcing people to delete more text in every PR is toil / paperwork and I already feel bad for introducing it.

I'm going to leave the bulk of this description commented out, including the policy link, and only uncomment the jira link.

This way if it gets deleted, @ripplekhera will know you intentionally did not want to associate a ticket.

This is what you will see in the PR description box:

```
# Proposed Changes

<!--
Hi there. Thanks for submitting a pull request!

Please describe your changes in this PR description.
You can also include a link to an active Jira ticket that provides relevant context if there is one.

https://zephyrai.atlassian.net/wiki/spaces/HELIX/pages/37650458/Secure+Software+Development+and+Product+Security
-->

**Jira ticket (optional)**: https://zephyrai.atlassian.net/browse/TICKET

Describe your changes.
```

And this is what it will look like.

# Proposed Changes

**Jira ticket (optional)**: https://zephyrai.atlassian.net/browse/TICKET

Describe your changes.

### CODE REVIEW

**Review by Jiafi — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
 
 Please describe your changes in this PR description.
 You can also include a link to an active Jira ticket that provides relevant context if there is one.
 
-https://policies.zephyrai.bio/sdlc.html
+https://zephyrai.atlassian.net/wiki/spaces/HELIX/pages/37650458/Secure+Software+Development+and+Product+Security
+-->
 
-# Proposed Changes
+**Jira ticket (optional)**: https://zephyrai.atlassian.net/browse/TICKET
```
Comment (kfreisen) [inline .github/PULL_REQUEST_TEMPLATE.md:12]: do we need `optional` here? If it really is optional you can just delete the line

**Review by kfreisen — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (jrouly) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: It's probably redundant, but I like that it's explicit.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,10 +1,14 @@
+# Proposed Changes
+
+<!--
 Hi there. Thanks for submitting a pull request!
 
 Please describe your changes in this PR description.
 You can also include a link to an active Jira ticket that provides relevant context if there is one.
 
-https://policies.zephyrai.bio/sdlc.html
+https://zephyrai.atlassian.net/wiki/spaces/HELIX/pages/37650458/Secure+Software+Development+and+Product+Security
```
Comment (arkban) [inline .github/PULL_REQUEST_TEMPLATE.md:9]: NIT: This should probably link to a more basic page, which then links to the security page.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,10 +1,14 @@
+# Proposed Changes
+
+<!--
 Hi there. Thanks for submitting a pull request!
```
Comment (arkban) [inline .github/PULL_REQUEST_TEMPLATE.md:4]: You missed some boilerplate.

Comment (jrouly) [reply .github/PULL_REQUEST_TEMPLATE.md:4]: This boilerplate is commented out markdown. It lives in the PR template, but it won't show up in PR descriptions and for that reason I don't consider it technically boilerplate -- you don't actually have to delete it or mess with it. It's there for the author's consideration only.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (jrouly) [reply .github/PULL_REQUEST_TEMPLATE.md:9]: This is our SDLC process documentation. I replaced the dead link to `policies.zephyrai.bio` with a live link to our internal wiki.

**Pull request merged**